### ISSUE TITLE
Stuff doesn't work

### ISSUE THREAD

**Issue by StFS** [sparkbox/lunch-bot#17]
Hi. I just tried deploying this to my personal heroku account and it doesn't seem to work.

Sorry for the extremely unhelpful title and probably extremely dumb questions here. I have no experience with neither Heroku nor Node.js so I'm hoping that his is just something really silly that somebody can help me with really quickly.

But basically, after following the instructions in the README, deploying to heroku and setting up a google spreadsheet document and all that, I get an error when I try to open the app through the heroku web console.

Looking at the logs all I see is this:

```
2017-01-12T14:34:41.630213+00:00 app[web.1]: npm ERR! node v6.9.1
2017-01-12T14:34:41.632731+00:00 app[web.1]: npm ERR! missing script: start
2017-01-12T14:34:41.631827+00:00 app[web.1]: 
2017-01-12T14:34:41.632992+00:00 app[web.1]: npm ERR! If you need help, you may report this error at:
2017-01-12T14:34:41.641753+00:00 app[web.1]: 
2017-01-12T14:34:41.641903+00:00 app[web.1]: npm ERR! Please include the following file with any support request:
2017-01-12T14:34:41.632912+00:00 app[web.1]: npm ERR! 
2017-01-12T14:34:41.642060+00:00 app[web.1]: npm ERR!     /app/npm-debug.log
2017-01-12T14:34:41.633060+00:00 app[web.1]: npm ERR!     <https://github.com/npm/npm/issues>
2017-01-12T14:34:42.915691+00:00 heroku[router]: at=error code=H10 desc="App crashed" method=GET path="/" host=nextcode-slack-lunch-duties.herokuapp.com request_id=198fee58-a2ad-405c-be9e-90a9f18a2dfa fwd="redacted" dyno= connect= service= status=503 bytes=
2017-01-12T14:34:44.257820+00:00 heroku[router]: at=error code=H10 desc="App crashed" method=GET path="/favicon.ico" host=nextcode-slack-lunch-duties.herokuapp.com request_id=fd96a530-1b7f-4134-a9b0-578bb548e0a8 fwd="redacted" dyno= connect= service= status=503 bytes=
```

**Comment by asimpson** [comment]
**TL;DR** The app doesn't have a `start` script because [Scheduler](https://devcenter.heroku.com/articles/scheduler) is the trigger.

@StFS sorry for the confusion. I haven't done a good job documenting the secret sauce to lunch-bot, [Heroku Scheduler](https://devcenter.heroku.com/articles/scheduler). 
![napkin 01-12-17 10 09 53 am](https://cloud.githubusercontent.com/assets/1048831/21895037/431fb2fc-d8af-11e6-9c6e-b2460849d1f2.png)

Scheduler runs the `general` script and the `private` script daily. Inside those scripts [lunch-bot checks if it's the correct day](https://github.com/sparkbox/lunch-bot/blob/master/src/privateReminder.js#L40) to remind people. 

Does that help? Let me know if you're still getting errors.

**Comment by StFS** [comment]
Thanks for the quick feedback. This definitely helps but just to be clear (as I have no experience as stated before):

1) Do I need to manually create these tasks in the scheduler after setting up according to the README.
2) Is it normal that I get these "App crashed" messages when I try to go to the web page (basically, there is no front end to this app) or will that be fixed once I set up the scheduler?

Thanks a bunch!

**Comment by asimpson** [comment]
1. Yes, you need to create the tasks in the scheduler. The actual task file in the command `node dist/generalAnnouncement.js` is generated by Heroku [after it installs everything](https://github.com/sparkbox/lunch-bot/blob/master/package.json#L14) from `npm`.

2. Yes, you will get an application error when you try to visit the app via the web browser. There is no front end, the only interface is the Slack messages that the bot posts.

**Comment by StFS** [comment]
Thank you!
